### PR TITLE
working on upload

### PR DESCRIPTION
**Pull request by ToosdaiOtte**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,6 +13,81 @@
...
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+    </div>
+    <div>
+      <img class="logo" src="https://tk-assets.lambdaschool.com/c7e8c14d-d7ad-4d21-9c18-4a68c77e5ad5_logo.png" alt="Great Idea! Company logo.">
+    </div>
+    </header>
```
Comment (jacobAngulo) [inline index.html:31]: it's never necessary to have something wrapping a single tag, the divs in this header could be removed

Comment (ToosdaiOtte) [reply index.html:31]: Makes sense, fixed it! Thank you for letting me know!

Comment (jacobAngulo) [reply index.html:31]: 🔥

File: index.html [hunk 2]
```diff
@@ -11,6 +13,79 @@
...
+        </div>
+      </div>
+  
+    </section>
+  
+    <section class="contact">
+  
+      <h4>Contact</h4>
+  
+      <p>123 Way 456 Street<br> Somewhere, USA</p>
```
Comment (jacobAngulo) [inline index.html:77]: address tag!

File: index.html [hunk 3]
```diff
@@ -11,6 +13,79 @@
...
+  
+      <p>sales@greatidea.io</p>
+  
+    </section>
+  
+    <footer>
+      <p>Copyright Great Idea! 2018</p>
+    </footer>
+  
+  </div><!-- container -->
```
Comment (jacobAngulo) [inline index.html:89]: all in all really organized and looking really good!

File: css/index.css [hunk 4]
```diff
@@ -1,3 +1,376 @@
...
+  }
+
+
+/* header */
+header {
+    display: flex;
+    justify-items: space-between;
+}
+
+@media(max-width: 800px) {
```
Comment (jacobAngulo) [inline css/index.css:86]: so stoked to see you doing media queries below their elements 🔥

**Review by jacobAngulo — APPROVED** [review]
good work Toosdai!

**Pull request merged**